pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -90,6 +90,12 @@ type MigPlanSpec struct {
 
 	// If set True, disables direct volume migrations.
 	IndirectVolumeMigration bool `json:"indirectVolumeMigration,omitempty"`
+
+	// IncludedResources optional list of included resources in Velero Backup
+	IncludedResources []*metav1.GroupKind `json:"includedResources,omitempty"`
+
+	// LabelSelector optional label selector on the included resources in Velero Backup
+	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty"`
```

Comment:
These must be optional as well, I would also document what happens when this is not added.